library/src/main/java/org/apache/apex/malhar/lib/state/managed/Bucket.java

```diff
@@ -353,6 +362,41 @@ public Slice get(Slice key, long timeBucket, ReadSource readSource)
       }
     }
 
+
+    private int filteredCount = 0;
+    private int unfilteredCount = 0;
+    private long verifyBloomFilterBeginTime = 0;
+    private long verifyBloomFilterInterval = 5000;
+    /**
+     * Test the result of using bloom filter and remove it if it not helpful for improving the performance.
+     *
+     * @param mightContain true if collection might contain the value
+     */
+    private void verifyBloomFilter(boolean mightContain)
```

Comment:
expectedFalsePositiveProbability() depends on expectedNumberOfFilterElements, which we in fact have no idea at all. At it is a fix value instead of dynamic value